### ISSUE TITLE
Images don’t have alt text

### ISSUE THREAD

**Issue by Diyap235** [SurajSG23/Job-Portal#18]
Description:
Some image tags are missing the alt attribute. This affects accessibility for screen readers .

Fix:
Add proper alt text to all <img> tags.

**Comment by Khwaaish** [comment]
please assign this issue to me

**Comment by Raj-Dwivedi2005** [comment]
"Hi! I'm a GSSoC'25 contributor and really interested in this issue. I'd love to work on it — could you please assign it to me? Thanks!"

**Comment by Samiksha-bajoria** [comment]
can u pls assign me this issue
